### PR TITLE
Fix couple things in docblocks

### PR DESCRIPTION
**Pull request by JeroenSormani**
Some typo's / added descriptions on some params/return / other fixes

### CODE REVIEW

File: includes/payments/functions.php [hunk 1]
```diff
@@ -1474,7 +1477,7 @@ function edd_get_payment_notes( $payment_id = 0, $search = '' ) {
 	}
 
 	remove_action( 'pre_get_comments', 'edd_hide_payment_notes', 10 );
-	remove_filter( 'comments_clauses', 'edd_hide_payment_notes_pre_41', 10, 2 );
```
Comment (chriscct7) [inline includes/payments/functions.php]: why was the 2 removed?

**Comment by chriscct7** [comment]
@JeroenSormani looks good. One little issue to fix (left a note)

File: includes/payments/functions.php (hunk 1 above)
Comment (JeroenSormani) [inline includes/payments/functions.php]: `remove_filter` doesn't have a 4th parameter ;-)

Comment (chriscct7) [inline includes/payments/functions.php]: oh lol, I didn't look at the remove part. Its all good

**Pull request merged**